cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/ActionType.java

```diff
@@ -12,24 +12,25 @@
  * Flags to indicate the type of an action.
  *
  * <ul>
- * <li>{@link #REPLICA_MOVEMENT}: Move a replica from a source broker to a destination broker.</li>
+ * <li>{@link #INTER_BROKER_REPLICA_MOVEMENT}: Move a replica from a source broker to a destination broker.</li>
+ * <li>{@link #INTRA_BROKER_REPLICA_MOVEMENT}: Move a replica from a source disk to a destination disk.</li>
  * <li>{@link #LEADERSHIP_MOVEMENT}: Move leadership of a leader from a source broker to a follower of the same
  * partition residing in a destination broker.</li>
- * <li>{@link #REPLICA_ADDITION}: Add a new replica to the cluster.</li>
- * <li>{@link #REPLICA_DELETION}: Remove an existing replica from the cluster.</li>
- * <li>{@link #REPLICA_SWAP}: Swap places of replicas residing in source and destination brokers.</li>
+ * <li>{@link #INTER_BROKER_REPLICA_SWAP}: Swap places of replicas residing in source and destination brokers.</li>
+ * <li>{@link #INTRA_BROKER_REPLICA_SWAP}: Swap places of replicas residing in source and destination disks.</li>
  * </ul>
  */
 public enum ActionType {
-  REPLICA_MOVEMENT("REPLICA"),
+  INTER_BROKER_REPLICA_MOVEMENT("REPLICA"),
```

Comment:
Nit: Ideally it is preferable to make such a global name change in a patch that comes before this one. Hence, we make the changes in this large patch easier to parse.